.tool-versions

```diff
@@ -1 +1,2 @@
 terraform 1.6.1
+trivy 0.48.0
```

Comment:
end of the line should be added here, you can setup your editor to always insert EOL for edited files

Reply:
Added in [b54e859](https://github.com/nvminhtue/tyson_devops_ic/pull/22/commits/b54e859d0c392fe406837be215791fb55b00bbe5)